pkg/idempotent/redis/redis.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+// redis is a simple inmemory Recorder implementation. This implementation
+// is intended for simple usecases (local dev) and not production workloads.
+package redis
+
+import (
+	"context"
+
+	redis "github.com/gomodule/redigo/redis"
+)
+
+var (
+	// Used for Redis value
+	defaultAddress = "localhost:6379"
```

Comment:
Can we override this with a `REDIS_INSTANCE` environment variable? If empty then use the localhost value.